ortho_seq_code/orthogonal_polynomial.py

```diff
@@ -6,26 +6,13 @@
 import pandas as pd
 import ortho_seq_code.sr as sr
 from ortho_seq_code.constants_orthoseqs import *
-from ortho_seq_code.utils import get_seq_info
+from ortho_seq_code.utils import *
```

Comment:
import * is usually not recommended. If you don't want to list out all functions but you need all the functions in this module. It would better if you could do import ortho_seq_code.utils as utils and use the imported functions by calling utisl.get_something etc

https://stackoverflow.com/questions/2386714/why-is-import-bad